DasharoPayloadPkg/GraphicsOutputDxe/GraphicsOutput.c

```diff
@@ -409,6 +409,10 @@ GraphicsOutputDriverBindingStart (
               if (Resources->AddrRangeMin == GraphicsInfo->FrameBufferBase) {
                 FrameBufferBase = Resources->AddrRangeMin;
                 break;
+              } else if (Resources->AddrRangeMin <= GraphicsInfo->FrameBufferBase

```

Comment:
I think the new `if`-statement can just replace the old one (new condition is a superset of the old one) to make the code simpler.

Reply:
Okay, removed the old `if` in commit beba2f7540ab56537875bda1f6b8acc8bad9af5b